frontend/src/app/modules/members/members-page/members-page.component.ts

```diff
@@ -61,6 +62,9 @@ export class MembersPageComponent extends BaseComponent implements OnInit {
             .pipe(this.untilThis)
             .subscribe(roles => {
                 this.roles = roles;
+                if (this.roles[0].name === 'Owner') {
```

Comment:
Could you please rewrite this part? 
If we don't need this role in the frontend at all then filter it on the backend.
If we need it then use .filter function here

Reply:
Fix, use filter here